internal/controller/apiextensions/composite/composed.go

```diff
@@ -360,10 +361,11 @@ func (r *APIDryRunRenderer) Render(ctx context.Context, cp resource.Composite, c
 		SetCompositionResourceName(cd, *t.Name)
 	}
 
-	// We do this last to ensure that a Composition cannot influence owner (and
-	// especially controller) references.
```

Comment:
This comment (which I'm pretty sure I wrote) make it sound like I actively didn't want a Composition to be able to set owner references. I can't remember why, though. 🤔

Reply:
I can see where you would be concerned about a user constructing an ownerReference that would interfere with the Crossplane controller's ownerReference, so it made sense to me that you might want to block it if no one was using it.